coalib/collecting/Collectors.py

```diff
@@ -170,7 +170,9 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
 
     if warn_if_unused_glob:
         _warn_if_unused_glob(log_printer, bear_globs, bear_globs_with_bears,
-                             "No bears were found matching '{}'.")
+                             "No bears were found matching '{}'. Make sure you "
+                             "typed the name correctly or install it from here "
+                             "https://github.com/coala-analyzer/coala-bears.")
```

Comment:
He should, but my tool is in no position of being completely usable now, its purely experimental and still has the import issue, so I'd rather not mention it anywhere yet. Once we decide to integrate it, I'll do all the necessary changes to point to the tool everywhere in coala where its needed :)